src/auditor/auditor_pallet.py

```diff
@@ -0,0 +1,18 @@
+"""
+auditor_pallet.py: forklift-compliant entry point
+"""
+from forklift.models import Pallet
+
+from . import auditor, credentials
+
+
+class AuditorPallet(Pallet):
+
+    def requires_processing(self):
```

Comment:
**question**: should this have a date check and only run on certain days?

Reply:
Probably. I still need to flesh out the full forklift integration; for this PR, I just want to get the file and logic skeleton in place.